keyboards/atreus62/info.json

```diff
@@ -5,8 +5,9 @@
   "width": 15,
```

Comment:
This file is configured for Atreus62. If this is meant for the EP40, this file needs to be at:

`keyboards/handwired/ep40/info.json`